pkg/ingester/ingester.go

```diff
@@ -98,7 +98,8 @@ type Config struct {
 	DefaultLimits    InstanceLimits         `yaml:"instance_limits"`
 	InstanceLimitsFn func() *InstanceLimits `yaml:"-"`
 
-	// For testing, you can override the address and ID of this ingester.
+	// For testing, you can over:
+	//ride the address and ID of this ingester.
```

Comment:
typo?

Reply:
Not only it's a typo, it is an accidental file change. I will revert change in `ingester.go` thanks